setup-homebrew/main.sh

```diff
@@ -28,30 +28,10 @@ function git_retry {
   return 0
 }
 
-# Clone Homebrew/brew and Homebrew/linuxbrew-core if necessary.
-if ! which brew &>/dev/null; then
```

Comment:
Could consider keeping this check around and exiting if it fails?

Reply:
Done